.github/workflows/python.yml

```diff
@@ -37,18 +37,16 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        # We pin black's version until either PyPy3 becomes Python 3.8 or black
-        # removes its typed-ast dependency. See
-        # https://github.com/psf/black/issues/727
-        pip install codecov black==19.3b0
```

Comment:
codecov appears to be not installed in the new changes.

Reply:
Looks like `pip install .[dev]` does it?
```
    "dev": ["coverage", "flake8", "pytest >= 4.6", "pytest-cov", "sphinx"],
```
coverage? pytest-cov?

Reply:
> Looks like `pip install .[dev]` does it?

Yes, coverage is already included in [dev], so it was redundant to install it separately.

> coverage? pytest-cov?

Not sure what you want to comment here. Please try to formulate sentences, it makes communication much clearer and easier for everyone.

Reply:
> coverage? pytest-cov?

I just wasn't sure which of the packages in "dev" does the codecov install, since coverage/pytest-cov is not codecov, but probably has codecov as a dependency.

Reply:
It was already clear from the context what I was referring to when I said the module names with question marks.

Reply:
My point is: don't unnecessarily accuse people.

Reply:
It is hard to determine tone with just text. I don't think @Corvince was being accusatory. I didn't understand what you were referring to either. This was a misunderstanding.

Reply:
Since more than one person were not able to understand what I said, then it is my mistake. I am sorry.